server/README.md

```diff
@@ -254,8 +254,47 @@ cargo run --bin remote_server_cli -- initialise-from-central -u 'user1:password1
 cargo run --bin remote_server_cli -- refresh-dates
 ```
 
-## Logging
+# Logging
 
 By default, the server logs to console with a logging level of `Info`
 You can configure this, to log to a file, for example, with a rollover of log files based on file size.
-See the `example.yaml` file for the available options.
\ No newline at end of file
+See the `example.yaml` file for the available options.
+
+# Directory structure
+
+An overview of how files are organised in the server repo is shown below, with annotations as noted.
+The files have been split in this way, grouped in crates, partly to improve build times - as well as a logical organisation of grouped concerns.
+
+```
+server
+├─ android (hoisting for the android implementation)
+├─ cli
+├─ configuration (runtime configuration files)
+├─ data (contains reference data which can be used by the cli to initialise the database)
+├─ docker (a zola docker instance)
```

Comment:
not sure if this is used?

Reply:
not anymore, I think we can delete it for now (can always fish it out from history)